public/data_load/kibana_cloned_code/courier.ts

```diff
@@ -116,9 +121,10 @@ export const handleCourierRequest = async ({
   // since the last request
   const shouldQuery = forceFetch || (searchSource as any).lastQuery !== queryHash;
 
+  let request;
   if (shouldQuery) {
     inspectorAdapters.requests.reset();
-    const request = inspectorAdapters.requests.start(
```

Comment:
Why this change?
you use  `request` only in this `if` block, and you instanciate `request` variable only one time.
So I think that it is better to keep this code as before.

Reply:
I made this change while updating the courier.ts file from the esaggs module in Kibana 7.12.1 while searching what was the problem with the data inspector not working. In the esaggs module is the `request`variable is instantiated this way and that's why I made the change.

I have reverted the change as requested